lucille-core/src/main/java/com/kmwllc/lucille/core/Runner.java

```diff
@@ -165,25 +165,24 @@ public static Map<String, TestMessenger> runInTestMode(String configName) throws
   private static Map<String, List<Exception>> logValidation(Config config) throws Exception {
     Map<String, List<Exception>> exceptions = validatePipelines(config);
 
-    for (Map.Entry<String, List<Exception>> entry : exceptions.entrySet()) {
-      if (!entry.getValue().isEmpty()) {
-        StringBuilder message =
-            new StringBuilder("Configuration is invalid. " + "Printing the list of exceptions for each pipeline\n");
-
-        for (Map.Entry<String, List<Exception>> entryInner : exceptions.entrySet()) {
-          message.append("\tConnector: ").append(entryInner.getKey()).append("\tError count: ").append(entryInner.getValue().size())
-              .append("\n");
-          int i = 1;
-          for (Exception e : entryInner.getValue()) {
-            message.append("\t\tException ").append(i++).append(": ").append(e.getMessage()).append("\n");
-          }
+    if(!exceptions.entrySet().stream().allMatch(e -> e.getValue().isEmpty())) {
+      StringBuilder message =
+      new StringBuilder("Configuration is invalid. " + "Printing the list of exceptions for each pipeline\n");
+
+      for (Map.Entry<String, List<Exception>> entryInner : exceptions.entrySet()) {
+        message.append("\tConnector: ").append(entryInner.getKey()).append("\tError count: ").append(entryInner.getValue().size())
+          .append("\n");
+        int i = 1;
+        for (Exception e : entryInner.getValue()) {
+          message.append("\t\tException ").append(i++).append(": ").append(e.getMessage()).append("\n");
         }
-
-        log.error(message.delete(message.length() - 1, message.length()).toString());
-        return exceptions; 
       }
+
+      log.error(message.delete(message.length() - 1, message.length()).toString());
+
+    } else {
```

Comment:
when you have an if/else where one clause is really simple, it can help readability to put that first (as part of the the if)

In this case, you could take the ! out of `if(!exceptions.entrySet(...` and put the "Configuration is valid" message there

also note that per the java conventions we follow, there should be a space between an "if" and the open paren